products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md

```diff
@@ -1,76 +1,70 @@
- + Why/when do Veterans seek out telehealth or telemedicine appointments? 
- + What types of appointments do Veterans consider using telehealth appointments for? 
- + Where would Veterans look to schedule/access this type of same day care (phrasing: How would you make a telehealth appointment? To be seen the same day?)
- + Are Veterans willing/able to use online methods for same day care
- + What devices/platforms do you have access to? Are you familiar with or do you currently use any video conferencing apps? If so, which ones? Are you familiar with any from VA?
- + What type of devices/platforms have Veterans used to receive similar type of access (if at all?)
- + Where can we contextually redirect Veterans to appropriate access points based on need (e.g., if immediate need vs future facing need such as scheduled appointment)
- + What (if any) of the existing digital tools would a Veteran assume they'd use to receive immediate, same day digital care?
- + Is "same-day" care the same thing as scheduling an appointment? If not, how is this different?
- + Have you ever made a same-day appointment
- + Explain your current process for getting a same-day appointment
- + What is the latest time of day a Veteran should be able to get "same-day" care? // *I think this is a business question as written, but we should ask Veterans about when they expect they should be seen for same day visits*
- + We do not want to show express care/same day options to Veterans who do not have access to it. Right now there are a lot of rules around who can use existing "express care." Technical research - are there ways to conditionally show this potential care option to Veterans who are eligible to access it? // *we could either show a bare bones wireframe or have them walk us through their experience currently: "Show us how you would make this appointment..."
+- Do Veterans consider "same-day" care the same thing as scheduling an appointment? If not, how is this different?
+- What medical problems/types of appointments do Veterans consider using Telehealth appointments for?
+- How do Veterans define "Express Care"? Is it Telehealth or in-person?
+- How do Veterans seek same-day / urgent care virtually? Can they do this currently through the VA?
+- What type of devices/platforms do Veterans use for video chats (if at all)?
+- Are Veterans comfortable getting medical advice via Telehealth? If so, what for? If not, why?
+- What VA health care resources are Veterans aware of related to COVID-19?
+- Will Veterans use a VA online COVID-19 screening tool?
+- Will Veterans choose Express Care for COVID-19 screening?
+- When do Veterans think COVID-19 Express Care appointments be available to them?
 
+## Timeline:
+- ***Friday, 4/3 have researched/tested options for rapid implementation within VAOS on directing Veterans to immediate screening and/or care due to COVID-19***
```

Comment:
I would clarify here that it's being tested with 'legacy' VAOS, not VAOS-R (new VAOS on VA.gov)

Reply:
yes, and that the timeline is undetermined ATM.